packages/lib/zuauth/test/zuauth.spec.ts

```diff
@@ -192,14 +195,16 @@ describe("zuauth should work", async function () {
           publicKey
         }
       ]);
-      assert(false, "Should not reach this point due to exception");
     } catch (e) {
-      expect(e).to.exist;
+      expect(e instanceof ZuAuthAuthenticationError).to.be.true;
+      thrown = true;
     }
+    expect(thrown).to.be.true;
   });
 
   it("should not authenticate PCDs with the wrong event ID", async function () {
```

Comment:
Why is this test case not new or modified?  It seems like this test case is supposed to be testing the eventID check, but was already passing when that check was missing.  That suggests to me that there might be something wrong with the test, which was causing it to throw for some other reason.  Is that why you added the new ZuAuthAuthenticationError type?  I think it may be worth doing a check on the error message to make sure you're getting an exception specifically from the case you expect, vs. accidentally triggering a public key check instead, or something similar.

Reply:
The original test was broken and this one isn't.